xml_inscriptions/bib_only/MA.Bos.MFA.G.01.7515.xml

```diff
@@ -35,12 +36,65 @@
                </msIdentifier>
                <msContents>
                   <textLang mainLang="grc" otherLangs=""/>
-                  <msItem>
-                     <p/>
+                  <msItem class="#text_unknown">
+                     <p>text on temple key in the shape of a bearded snake </p>
                   </msItem>
                </msContents>
-               <physDesc><!--objectDesc, handDesc, decoDesc go here--></physDesc>
-               <history><!--summary, origin, provenance go here--></history>
+               <!--  *************************************  <physDesc>  ********************************* -->
+               <physDesc>
+                  <!--  *************************************  <objectDesc>  *********************************  -->
+                  <objectDesc ana="#fragment #object_other">
+                     <supportDesc ana="#metal.bronze">
+                        <support>
+                           <p> bronze temple key </p>
+                           <dimensions type="surface" unit="cm">
+                              <height/>
+                              <width>39.6 </width>
+                              <depth/>
+                           </dimensions>
+                        </support>
+                        <condition ana="#complete.broken">
+                           <p> </p>
+                        </condition>
+                     </supportDesc>
+                     <!--  *************************************  <layoutDesc>  ********************************* -->
+                     <layoutDesc>
+                        <layout columns="0" writtenLines="1">
+                           <p/>
+                        </layout>
+                     </layoutDesc>
+                  </objectDesc>
+                  <!--  *************************************  <handDesc>  ********************************* -->
+                  <handDesc hands="1">
+                     <handNote ana="#inscribed"> "incised" </handNote>
+                  </handDesc>
+                  <!--  *************************************  <decoDesc>  *********************************     -->
+                  <decoDesc>
+                     <decoNote>
+                        <p/>
+                     </decoNote>
+                  </decoDesc>
+               </physDesc>
+               <!--  *************************************  <history>  *********************************  -->
+               <history>
+                  <summary>
+                     <rs/>
+                     <!-- content can be "modern" or "modern-text". A blank value indicates that the inscription is ancient.  -->
+                  </summary>
+                  <origin>
+                     <date evidence="#unspecified" notBefore="-0499" notAfter="-0476">about 500-475
+                        B.C.</date>
+                     <placeName ref="europe.greece.athens"/>
```

Comment:
Put the actual name of the place here -- I suspect it's not "Athens" given what you say in the first provenance item.